### PR TITLE
firstParty flag for Ruby

### PR DESCRIPTION
**Pull request by haidong**
To address #4795, this is part of a series of PRs that add isFirstParty or isNative flags to instrumentation registries that lack them.

This batch touches Ruby instrumentation registries.

### CODE REVIEW

File: data/registry/instrumentation-ruby-elasticsearch.yml [hunk 1]
```diff
@@ -21,3 +21,4 @@ package:
   registry: gems
   name: elasticsearch
   version: 8.15.0
+isFirstParty: true
```
Comment (svrnm) [inline data/registry/instrumentation-ruby-elasticsearch.yml:24]:
```suggestion
isNative: true
```

**Review by svrnm — CHANGES_REQUESTED** [review]

**Review by svrnm — APPROVED** [review]

**Comment by theletterf** [comment]
@svrnm How can we fix those registry errors?

**Comment by svrnm** [comment]
> @svrnm How can we fix those registry errors?

there is a real error in one of the files:

[07:22:09] YAMLException: duplicated mapping key (24:1)

File: data/registry/instrumentation-ruby-elasticsearch.yml [hunk 2]
```diff
@@ -21,3 +21,4 @@ package:
   registry: gems
   name: elasticsearch
   version: 8.15.0
+isNative: true
```
Comment (svrnm) [inline data/registry/instrumentation-ruby-elasticsearch.yml:24]:
```suggestion
```

there is an `isNative` already in line 19

**Pull request merged**